src/transformers/models/instructblipvideo/processing_instructblipvideo.py

```diff
@@ -71,30 +93,24 @@ def __init__(self, image_processor, tokenizer, qformer_tokenizer=None, num_query
 
     def __call__(
         self,
-        images: VideoInput = None,
-        text: Union[TextInput, PreTokenizedInput, List[TextInput], List[PreTokenizedInput]] = None,
-        add_special_tokens: bool = True,
-        padding: Union[bool, str, PaddingStrategy] = False,
-        truncation: Union[bool, str, TruncationStrategy] = None,
-        max_length: Optional[int] = None,
-        stride: int = 0,
-        pad_to_multiple_of: Optional[int] = None,
-        return_attention_mask: Optional[bool] = None,
-        return_overflowing_tokens: bool = False,
-        return_special_tokens_mask: bool = False,
-        return_offsets_mapping: bool = False,
-        return_token_type_ids: bool = False,
-        return_length: bool = False,
-        verbose: bool = True,
-        return_tensors: Optional[Union[str, TensorType]] = None,
-        **kwargs,
+        images: Optional[VideoInput] = None,
+        text: Optional[Union[TextInput, PreTokenizedInput, List[TextInput], List[PreTokenizedInput]]] = None,
+        audio=None,
+        videos=None,
```

Comment:
@zucchini-nlp I've made these changes:
- we now allow the user to use `videos` instead of `images`
- using `images` now results in a future warning
- using both `videos` and `images` now results in an error